.env

```diff
@@ -1,6 +1,15 @@
-PORT = 3000
+PORT=3000
 POSTGRES_USER=polkastats
 POSTGRES_PASSWORD=polkastats
 POSTGRES_DATABASE=polkastats
 POSTGRES_HOST=localhost
 POSTGRES_PORT=5432
+
+#Environment variables for tests
+TESTS_WS_PROVIDER_URL='ws://host.docker.internal:9944'
+TESTS_UNIQUE_WS_ENDPOINT=wss://ws-quartz-dev.unique.network
+TESTS_TYPE_PROVIDER=quartz
+TESTS_GRAPHQL_URL='http://localhost:3031/graphql'
+TESTS_CHAIN_VERSION=918001
+TESTS_CRAWLER_API_REPOSITORY=https://github.com/UniqueNetwork/unique-explorer-api/
+TESTS_CRAWLER_BRANCH=v1.6.3
```

Comment:
At least these three variables would be redundant when you will fix your running scripts

Reply:
done